pandas/tests/resample/test_deprecated.py

```diff
@@ -0,0 +1,260 @@
+from datetime import datetime, timedelta
+
+import numpy as np
+import pytest
+
+import pandas as pd
+from pandas import DataFrame, Series
+import pandas._testing as tm
+from pandas.core.indexes.datetimes import date_range
+from pandas.core.indexes.period import PeriodIndex, period_range
+from pandas.core.indexes.timedeltas import TimedeltaIndex, timedelta_range
+
+from pandas.tseries.offsets import BDay, Minute
+
+DATE_RANGE = (date_range, "dti", datetime(2005, 1, 1), datetime(2005, 1, 10))
+PERIOD_RANGE = (period_range, "pi", datetime(2005, 1, 1), datetime(2005, 1, 10))
+TIMEDELTA_RANGE = (timedelta_range, "tdi", "1 day", "10 day")
+
+all_ts = pytest.mark.parametrize(
```

Comment:
This is neat; don't think we've done this elsewhere

Reply:
`all_ts` is a copy of `all_ts` from `pandas/tests/resample/test_base.py` :)
I just copied the test that will be deprecated in `test_deprecated.py` for an easier removal when we will reach pandas 2.0. :wink: